app/images/icons/collapse.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path d="m422 64c5 0 9 2 13 5 7 8 7 19 0 26l-107 107 66 0c11 0 19 7 19 18 0 12-8 19-19 19l-110 0c-2 0-6 0-8-2-2 0-3-2-5-4-2-1-4-3-4-5-2-2-2-6-2-8l0-110c0-11 8-19 19-19 11 0 18 8 18 19l0 66 107-107c4-3 7-5 13-5z m-194 193l-110 0c-11 0-19 8-19 19 0 11 8 18 19 18l66 0-107 107c-7 7-7 19 0 26 8 7 19 7 26 0l107-107 0 66c0 11 7 19 18 19 11 0 19-8 19-19l0-110c0-2 0-6-2-8 0-1-2-3-4-5-2-2-3-4-5-4-2-2-6-2-8-2z"/></svg>
```

Comment:
Just pushed a commit to optimize the svg here. https://github.com/MetaMask/metamask-extension/pull/28038/commits/44ec97d0b8139c0655a61c559ac8134ef3ab4fa7